### PR TITLE
Fixed typo, added readme

### PR DESCRIPTION
**Pull request by MaticBabnik**
Fixed your project, thank me later.

### CODE REVIEW

**Comment by marwuint** [comment]
Nice catch!

Can you also add some content in readme?

**Comment by MaticBabnik** [comment]
Added docs :)

xd

**Comment by gigibu5** [comment]
+1

**Comment by Axodus** [comment]
Add actual docs, then we will merge

File: package.json [hunk 1]
```diff
@@ -10,10 +10,10 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://gitlab.vegova.si/AikenAhac/vegova-bot.git"
+    "url": "https://github.com/VegovaLjubljana/vegova-bot.git"
   },
-  "author": "Aiken Tine Ahac, Martin Hanzlowsky",
-  "license": "ISC",
+  "author": "Aiken Tine Ahac, Martin Hanzlowsky, Matic Babnik",
+  "license": "MIT",
```
Comment (marwuint) [inline package.json:16]: 👍

File: readme.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+set TOKEN=<INSERT TOKEN HERE>
+```
+or create a `.env` file with the following content:
+```
+TOKEN=<INSERT TOKEN HERE>
+```
+
+## Running
+Run `npm start` from inside the folder.
+You can also deploy this repo to Heroku.
```
Comment (marwuint) [inline readme.md]: Nice, an actual basic readme!

**Review by marwuint — APPROVED** [review]

**Comment by Axodus** [comment]
Kek, looks good

**Pull request merged**